src/Skills.jsx

```diff
@@ -93,10 +94,27 @@ class Skills extends Component {
 
   componentDidMount() {
     console.log(process.env)
+    const currentId = (this.props.params) ? this.props.params.id : null;
     axios.get(api + `/skills/`)
       .then(res => {
         const skills = res.data.map(obj => obj);
         this.setState({ skills });
+        if (currentId) {
```

Comment:
This isn't going to work completely because a GET on `skills/` returns an array of skills without children. ` [{name, id, skill_type}, ...]` and a GET on `skills/{id}` returns a single skill with children {name, id, skill_type, links:[{link_name, link_id, url},...]}.  I would recommend breaking this functionality up.

Also, if you route to a url with an invalid skill_id we should be be displaying a not found message, like https://github.com/maryvilledev/skilldirectoryui/issues/349, instead of rerouting to skills/